service/src/main/kotlin/fi/espoo/evaka/placement/PlacementControllerCitizen.kt

```diff
@@ -146,10 +146,7 @@ class PlacementControllerCitizen(
                         }
 
                         val cancelableTransferApplicationIds =
-                            tx.cancelAllActiveTransferApplicationsAfterDate(
```

Comment:
Should this function be deleted too if it's no longer used?

Reply:
Actually it was renamed and refactored, but might be hard to see from this diff